arm-sql/2014-04-01/examples/DatabaseSecurityAlertCreateMin.json

```diff
@@ -0,0 +1,55 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-1111-2222-3333-444444444444",
+    "resourceGroupName": "securityalert-4799",
+    "serverName": "securityalert-6440",
+    "databaseName": "testdb",
+    "securityAlertPolicyName": "Default",
+    "api-version": "2014-04-01",
+    "databaseSecurityAlertPolicy": {
+      "properties": {
+        "state": "Enabled",
+        "storageAccountAccessKey": "sdlfkjabc+sdlfkjsdlkfsjdfLDKFTERLKFDFKLjsdfksjdflsdkfD2342309432849328476458/3RSD==",
+        "storageEndpoint": "https://mystorage.blob.core.windows.net"
+      }
+    }
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/00000000-1111-2222-3333-444444444444/resourceGroups/securityalert-4799/providers/Microsoft.Sql/servers/securityalert-6440/databases/testdb",
+        "name": "Default",
+        "type": "Microsoft.Sql/servers/databases/securityAlertPolicies",
+        "kind": "V12",
+          "properties": {
+            "state": "Enabled",
+            "emailAccountAdmins": "Enabled",
+            "emailAddresses": "",
+            "disabledAlerts": "",
```

Comment:
Actually the min/max examples are already accurate (when enabled=true). I just have to make state required.